samples/CommunityToolkit.Maui.Sample/Pages/Views/MediaElement/MediaElementMultipleWindowsPage.cs

```diff
@@ -20,17 +20,29 @@ public MediaElementMultipleWindowsPage(MediaElementMultipleWindowsViewModel view
 #if WINDOWS || MACCATALYST
 		secondWindow = new Window(new ContentPage
 		{
-			Content = new MediaElement
+			Content = new Grid
```

Comment:
Why are we putting a `MediaElement` inside a `Grid`? 

Layouts should only be used if multiple visual elements are required.

Reply:
Updated to remove `Grid` as requested